programs/disks/DisksApp.cpp

```diff
@@ -57,7 +57,7 @@ void DisksApp::addOptions(
         ("config-file,C", po::value<String>(), "Set config file")
         ("disk", po::value<String>(), "Set disk name")
         ("command_name", po::value<String>(), "Name for command to do")
-        ("send-logs", "Send logs")
+        ("save-logs", "Save logs to a file")
```

Comment:
Wouldn't it make sense to make the change backward compatible? i.e. keep current option but add `save-logs` with the same effect.

Reply:
It's totally fine to break compatibility in this particular case, because this feature is new and not widely used (moreover, it did not work, see https://github.com/ClickHouse/ClickHouse/issues/42260). Also because this incompatibility will not introduce much inconvenience and it's easy to workaround.